docs/usermanual/userman_admin.rst

```diff
@@ -645,14 +645,23 @@ You can also run the ``kalite`` with a completely different Python settings modu
 Changing base settings
 ^^^^^^^^^^^^^^^^^^^^^^
 
-By default, ``~/.kalite/settings.py`` will load ``kalite.project.settings.base`` which are the basic settings. But you can also load Raspberry Pi settings by changing the file to read something like:
+By default, ``~/.kalite/settings.py`` will load ``kalite.project.settings.base`` which are the basic settings. But you can also load Raspberry Pi settings by changing the file to read something like::
 
-  .. ::
-    from kalite.project.settings.raspberry_pi import *
-    # Put your settings here, e.g.
-    # MY_SETTING_VAR = 123
+  from kalite.project.settings.raspberry_pi import *
+  # Put your settings here, e.g.
+  # MY_SETTING_VAR = 123
 
 
+.. NOTE::
+    When changing ``CONTENT_ROOT``, you should also change your ``DATABASES`` **if you
+    have downloaded your own assessment items** and you want to keep the
+    read-only assessment_items database (~50 MB) together
+    with your other media contents (for portability). Example::
+      
+      from kalite.project.settings.base import *
+      CONTENT_ROOT = '/example'
+      DATABASES['assessment_items']['NAME'] = os.path.join(CONTENT_ROOT, 'assessmentitems.sqlite')
```

Comment:
This message has too much technical information, and the _if_ makes it seem optional. Plus I assume most users have no opinion about portability. I prefer this:
```rst
.. warning::
    When changing ``CONTENT_ROOT``, you **must** also change the ``DATABASES['assessment_items']['NAME']`` setting to ``os.path.join(CONTENT_ROOT, 'assessmentitems.sqlite')`` on a following line. See this example::

        from kalite.project.settings.base import *
        CONTENT_ROOT = '/example'
        DATABASES['assessment_items']['NAME'] = os.path.join(CONTENT_ROOT, 'assessmentitems.sqlite')
```